win32/GNUmakefile

```diff
@@ -1213,7 +1206,6 @@ CFG_VARS	=					\
 		"useperlio=$(USE_PERLIO)"		\
 		"use64bitint=$(USE_64_BIT_INT)"		\
 		"uselongdouble=$(USE_LONG_DOUBLE)"	\
-		"uselargefiles=$(USE_LARGE_FILES)"	\
```

Comment:
How does this affect `$Config{uselargefiles}`?

Reply:
This part of makefile overrides config values from `config.gc` (or `config.vc`), so after this change, `$Config{uselargefiles}` will be set to the value from that file.